data/gui/screens/online/online.stkgui

```diff
@@ -21,16 +21,16 @@
         <buttonbar id="menu_toprow" proportion="3" width="90%" align="center">
             <icon-button id="lan" width="128" height="128"
                          icon="gui/icons/menu_multi.png" focus_icon="gui/icons/menu_multi_focus.png"
-                         I18N="Networking menu button" text="Local networking" word_wrap="true"/>
+                         I18N="Networking menu button" text="Local Network" word_wrap="true"/>
             <icon-button id="wan" width="128" height="128"
                          icon="gui/icons/menu_online.png" focus_icon="gui/icons/menu_online_focus.png"
-                         I18N="Networking menu button" text="Global networking" word_wrap="true"/>
+                         I18N="Networking menu button" text="Internet" word_wrap="true"/>
             <icon-button id="enter-address" width="128" height="128"
                          icon="gui/icons/online/menu_quick_play.png" focus_icon="gui/icons/online/menu_quick_play_hover.png"
-                         I18N="Networking menu button" text="Enter server address" word_wrap="true"/>
+                         I18N="Networking menu button" text="Direct Connect" word_wrap="true"/>
```

Comment:
Connect is a verb, "Direct connection" seems more appropriate to me, don't you think so ?